renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -3915,6 +3915,9 @@ rdcarray<PixelModification> VulkanReplay::PixelHistory(rdcarray<EventUsage> even
   if(events.empty())
     return history;
 
+  if(m_pDriver->GetRenderState().graphics.pipeline == ResourceId())
```

Comment:
This is much too wide a net to cast. The pixel history should be able to run without any pipeline bound at the point it's invoked because it is obtaining the history of that resource across all previous events. This would cause the pixel history to fail even on normal vulkan applications if there happens to be no graphics pipeline bound - including if there is a compute pipeline bound.

If you want to disable it only for shader objects you should target that more specifically.

Reply:
This has been updated accordingly.